packages/core/src/types.ts

```diff
@@ -603,7 +603,9 @@ export type Plugin = {
  */
 export enum Clients {
     DISCORD = "discord",
-    DIRECT = "direct",
+// you can't specify this in characters
+// all characters are registered with this
+//    DIRECT = "direct",
```

Comment:
not sure if this was intentional

Reply:
it is. Direct is meant to always run now